cypress/support/elements/portal-report/activity-score-settings.js

```diff
@@ -0,0 +1,79 @@
+    const option = ["No score", "Manual", "Rubric", "MCQ Scored"];
+    cy.get('[class^=feedback-settings-modal--modalOption--]').eq(option.indexOf(setting)).find('[data-cy=feedback-settings-radio-button]').click();
+  }
+  getCancelButton() {
+    return cy.get('[data-cy=feedback-settings-modal-close-button]').eq(0);
+  }  
+  getSaveButton() {
+    return cy.get('[data-cy=feedback-settings-modal-close-button]').eq(1);
+  }
+  getActivityFeedbackScore() {
+    return cy.get('[class*=activity-feedback-score--activityFeedbackScore--]');
+  } 
+  verifyScoreNotDisplayedInRubricScoreHeader() {
+    cy.get('[class^=rubric-table--rubricScoreHeader--]').eq(0).should("not.contain", "3");
+    cy.get('[class^=rubric-table--rubricScoreHeader--]').eq(1).should("not.contain", "2");
+    cy.get('[class^=rubric-table--rubricScoreHeader--]').eq(2).should("not.contain", "1");
+  }
+  verifyScoreDisplayedInRubricScoreHeader() {
+    cy.get('[class^=rubric-table--rubricScoreHeader--]').eq(0).should("contain", "3");
+    cy.get('[class^=rubric-table--rubricScoreHeader--]').eq(1).should("contain", "2");
+    cy.get('[class^=rubric-table--rubricScoreHeader--]').eq(2).should("contain", "1");
+  }
+  selectRubricScore(table, row, score) {
+    cy.get('[class^=rubric-table--rubricTable--]').eq(table - 1).find('[class^=rubric-table--ratingsGroup--]').eq(row - 1).find('[data-cy=rating-radio-button]').eq(score - 1).click();
+  }
+  getRubricSummaryIcon() {
+    return cy.get('[data-cy=rubric-summary-icon]');
+  }
+  verifyRubricSummaryIcon(score, prop) {
+    const option = ["Proficient", "Developing", "Beginning"];
+    this.getRubricSummaryIcon().find('[class^=rubric-summary-icon--rubricSummaryIconRow--]').eq(0).find('div').eq(option.indexOf(score)).invoke("attr", "style").should("contain", prop);
+    this.getRubricSummaryIcon().find('[class^=rubric-summary-icon--rubricSummaryIconRow--]').eq(1).find('div').eq(option.indexOf(score)).invoke("attr", "style").should("contain", prop);
+  }
+  getRubricSummaryDetailsDialog() {
+    return cy.get('[data-cy=rubric-summary-modal]');
+  }
+  verifyScoreNotDisplayedInRubricSummaryDetailsDialogHeader() {
+    this.getRubricSummaryDetailsDialog().find('[class^=rubric-table--rubricScoreHeader--]').eq(0).should("not.contain", "3");
+    this.getRubricSummaryDetailsDialog().find('[class^=rubric-table--rubricScoreHeader--]').eq(1).should("not.contain", "2");
+    this.getRubricSummaryDetailsDialog().find('[class^=rubric-table--rubricScoreHeader--]').eq(2).should("not.contain", "1");
+  }
+  verifyScoreDisplayedInRubricSummaryDetailsDialogHeader() {
+    this.getRubricSummaryDetailsDialog().find('[class^=rubric-table--rubricScoreHeader--]').eq(0).should("contain", "3");
+    this.getRubricSummaryDetailsDialog().find('[class^=rubric-table--rubricScoreHeader--]').eq(1).should("contain", "2");
+    this.getRubricSummaryDetailsDialog().find('[class^=rubric-table--rubricScoreHeader--]').eq(2).should("contain", "1");
+  }
```

Comment:
I'm wondering if the use of a method like this would make the above lines more legible:
  verifyScoreDisplayInRubricSummaryDetailsDialogHeader(index, containOrNot, string) {
    this.getRubricSummaryDetailsDialog().find('[class^=rubric-table--rubricScoreHeader--]').eq(index).should(containOrNot, string);
  }
Then each of the lines above would be something like this:
  verifyScoreDisplayInRubricSummaryDetailsDialogHeader(0, "not.contain", "3");
(You could also make containOrNot a boolean rather than a string, perhaps.)
It's a close call: my suggestion actually increases the number of lines of code, but makes most of those lines easier to read, I think.